<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginDialog</class>
 <widget class="QDialog" name="loginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>531</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_login">
    <attribute name="title">
     <string>登入</string>
    </attribute>
    <widget class="QLabel" name="lb_username">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_password">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_username">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_password">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_login">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>240</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>登入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_clear">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>240</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_registe">
    <attribute name="title">
     <string>注册</string>
    </attribute>
    <widget class="QLabel" name="lb_registe_username">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_registe_password">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_registe_password">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_registe_username">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_registe">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>300</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_registe_clear">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>300</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_registe_password_again">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_registe_password_again">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>确认密码：</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>431</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QCheckBox" name="cb_all">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>全选</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_allnot">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>全不选</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_food">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>美食</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_funny">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>搞笑</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_energy">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>正能量</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_music">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>歌曲</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_video">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>影视</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_outside">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>户外</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_dance">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>50</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>舞蹈</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_edu">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>80</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>教育</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
